docs/software/media.md

```diff
@@ -1124,4 +1125,27 @@ A FOSS web interface media streaming server, including live TV, forked from Emby
 
 Source code: <https://github.com/jellyfin/jellyfin>
 
+## Snapcast client
+
+A FOSS multi room audio solution.
+
+![Snapcast logo](../assets/images/dietpi-software-media-snapcastclient.png){: width="600" height="341" loading="lazy"}
```

Comment:
The image dimensions should match the actual image aspect ratio. In this case I suggest giving it a width of `300` (px) (commonly used by us for logos), which then results in a higher of `48`.

Reply:
Thanks fixed